src/Animation/SpringScheduler.lua

```diff
@@ -36,6 +37,28 @@ function SpringScheduler.add(spring: Spring)
 	local damping = spring._damping
 	local speed = spring._speed
 
+	if spring._dampingIsState then
+		damping = damping:get(false)
+	end
+	if spring._speedIsState then
+		speed = speed:get(false)
+	end
+
+	if typeof(damping) ~= "number" then
+		logError("mistypedSpringDamping", typeof(damping))
```

Comment:
All of these logError calls are incorrect, as they forget to pass the errObj arg
```Lua
local function logError(messageID: string, errObj: Types.Error?, ...)
```
 they should be `logError("...", nil, ...)` t

Reply:
Good catch! I’ll update the code :)

Reply:
By "catch" you mean *pull these changes locally and then screw up a spring because I'm dumb*